contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV1.sol

```diff
@@ -452,6 +471,28 @@ contract MinterMerkleV1 is ReentrancyGuard, IFilteredMinterMerkleV0 {
         }
     }
 
+    /**
+     * @notice projectId => maximum invocations per allowlisted address. If a
+     * a value of 0 is returned, there is no limit on the number of mints per
+     * allowlisted address.
+     * Default behavior is limit 1 mint per address.
+     * This value can be changed at any time by the artist.
+     * @dev default value stated above must be updated if the value of
+     * CONFIG_USE_MAX_INVOCATIONS_PER_ADDRESS_OVERRIDE is changed.
+     */
+    function projectMaxInvocationsPerAddress(uint256 _projectId)
```

Comment:
I think we for convenience probably want to also provide a similar view method that takes in an address as an input and returns the # of remaining for a given specific address?

Reply:
I've added a new function `projectMaxInvocationsPerAddress`. It's a little... weird, just because we have to return a bool indicating if the project limits the qty of mints allowed per allowlisted address, then also has to return qty of mints remaining. I think still worth it though.

Reply:
This seems like the right approach to me–TY ser! 🙏